lib/app/routes/app_router.dart

```diff
@@ -855,8 +867,9 @@ class AppRouter {
       return Pages.generosityChallenge.path;
     }
 
-    if (navigatingPage == Pages.generosityChallenge.path) {
-      return Pages.generosityIntroduction.path;
+    if (navigatingPage == Pages.generosityChallenge.path &&
+        !GenerosityChallengeHelper.isActivated) {
+      return Pages.generosityChallengeIntroduction.path;
     }
```

Comment:
because this is now a subroute it needs to be `'${Pages.generosityChallenge.path}/${Pages.generosityChallengeIntroduction.path}'`

Reply:
according to the tests, subroutes are working with just a subroute part

Reply:
i am not getting correctly redirected using the current code:
![simulator_screenshot_7324FCEF-8A32-4758-B4D5-FB06B9221570](https://github.com/givtnl/givt-app/assets/43034761/4cdc1989-d2b6-4cb5-b28d-4926a6d990b9)
 if i update using the whole path then i do get to the correct page

Reply:
ohh, let me fix it then

Reply:
fixed

Reply:
I think using only the subroute works when redirecting by name, but this is using the path so it needs to be full 🤔

Reply:
changed based on talk with Daniela: show intro from challenge itself, instead of redirection